pkg/local_object_storage/writecache/persist.go

```diff
@@ -26,6 +26,9 @@ func (c *cache) persistLoop() {
 
 			start := time.Now()
 			c.persistObjects(m)
+			for i := range m {
```

Comment:
It is calculated at about 5 lines below, we need just to set it to 0 (again, with mutex).

Reply:
Please have review.